GliaWidgets/SecureConversations/SecureConversations.ChatWithTranscriptModel.swift

```diff
@@ -303,7 +303,7 @@ extension SecureConversations {
         private let downloader: FileDownloader
         let fileUploadListModel: SecureConversations.FileUploadListViewModel
 
-        private var messageText = "" {
+        private (set) var messageText = "" {
```

Comment:
I wonder why the whole messageText was private in the first place when in the welcome view model it is public. I am using the "publicness" of the messageText to test that whenever you change it, the view model correctly triggers the render props.

Reply:
I do not allow changing `messageText` externally here, but only reading from it for unit tests, so it instead of being fully inaccessible for test case it is now read-only.